pkg/controller/installation/defaults_test.go

```diff
@@ -75,17 +80,29 @@ var _ = Describe("Defaulting logic tests", func() {
 					},
 				},
 				CalicoNetwork: &operator.CalicoNetworkSpec{
-					IPPools: []operator.IPPool{{
-						CIDR:          "1.2.3.0/24",
-						Encapsulation: "IPIPCrossSubnet",
-						NATOutgoing:   "Enabled",
-						NodeSelector:  "has(thiskey)",
-					}},
+					IPPools: []operator.IPPool{
+						{
+							CIDR:          "1.2.3.0/24",
+							Encapsulation: "IPIPCrossSubnet",
+							NATOutgoing:   "Enabled",
+							NodeSelector:  "has(thiskey)",
+							BlockSize:     &twentySeven,
+						},
+						{
+							CIDR:          "fd00::0/64",
+							Encapsulation: "None",
+							NATOutgoing:   "Enabled",
+							NodeSelector:  "has(thiskey)",
+							BlockSize:     &twentySeven,
```

Comment:
hmmm, this should be an error I think.

Reply:
Ah yeah, but we're not running the validation code in this test (just defaulting logic).